docs/dotnet/dotnet-dns-metrics.md

```diff
@@ -19,19 +19,17 @@ This article defines semantic conventions for DNS metrics emitted by .NET.
 
 ### Metric: `dns.lookup.duration`
 
-this metric SHOULD be specified with
+This metric SHOULD be specified with
 [`ExplicitBucketBoundaries`](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.26.0/specification/metrics/api.md#instrument-advisory-parameters)
 of `[ 0.005, 0.01, 0.025, 0.05, 0.075, 0.1, 0.25, 0.5, 0.75, 1, 2.5, 5, 7.5, 10 ]`.
 
-<!-- semconv metric.dotnet.dns.lookup.duration(metric_table) -->
+<!-- Tables in this document are not auto-generated and are intentionally frozen in time. From the .NET perspective this metric and its attributes are stable till the next major version. They are still experimental in the OpenTelemetry. -->
```

Comment:
I assume the reason the paragraph is not simply replaced with a reference to the common metric (like with `http.client.duration`) is the fact that the OTel definition is unstable? Is the plan to change this later?